acl_loader/main.py

```diff
@@ -114,12 +116,43 @@ def __init__(self):
         self.tables_db_info = {}
         self.rules_db_info = {}
         self.rules_info = {}
+        """
+           Load global db config. This call is no-op in single npu platforms
+        """
+        SonicDBConfig.load_sonic_global_db_config()
         self.sessions_db_info = {}
         self.configdb = ConfigDBConnector()
         self.configdb.connect()
         self.statedb = SonicV2Connector(host="127.0.0.1")
         self.statedb.connect(self.statedb.STATE_DB)
 
+        """ 
+            For multi-npu architecture we will have both gobal and per asic namespace.
+            Global namespace will be useful only for Control plane ACL which are via IPTables.
+            Per ASIC namespace will be used for Data and Everflow ACL's.
+            Global Configdb will have all ACL information and will be used as souurce of truth
+            for ACL modification to config DB which will be done to both Global DB and per asic namespace
+            corresponding to fron-end asic namespace only.
+        """
+        self.per_npu_configdb = {}
```

Comment:
Add empty line after this line for visual separation.

Reply:
@jleveque  Done